src/containers/Tenant/Diagnostics/DiagnosticsPages.ts

```diff
@@ -105,22 +109,23 @@ const DATABASE_PAGES = [
     network,
     describe,
     configs,
+    access,
     operations,
 ];
 
-const TABLE_PAGES = [overview, schema, topShards, nodes, graph, tablets, hotKeys, describe];
-const COLUMN_TABLE_PAGES = [overview, schema, topShards, nodes, tablets, describe];
+const TABLE_PAGES = [overview, schema, topShards, nodes, graph, tablets, hotKeys, describe, access];
+const COLUMN_TABLE_PAGES = [overview, schema, topShards, nodes, tablets, describe, access];
 
-const DIR_PAGES = [overview, topShards, nodes, describe];
+const DIR_PAGES = [overview, topShards, nodes, describe, access];
 
-const CDC_STREAM_PAGES = [overview, consumers, partitions, nodes, describe];
-const CDC_STREAM_IMPL_PAGES = [overview, nodes, tablets, describe];
-const TOPIC_PAGES = [overview, consumers, partitions, topicData, nodes, tablets, describe];
+const CDC_STREAM_PAGES = [overview, consumers, partitions, nodes, describe, access];
+const CDC_STREAM_IMPL_PAGES = [overview, nodes, tablets, describe, access];
+const TOPIC_PAGES = [overview, consumers, partitions, topicData, nodes, tablets, describe, access];
 
-const EXTERNAL_DATA_SOURCE_PAGES = [overview, describe];
-const EXTERNAL_TABLE_PAGES = [overview, schema, describe];
+const EXTERNAL_DATA_SOURCE_PAGES = [overview, describe, access];
+const EXTERNAL_TABLE_PAGES = [overview, schema, describe, access];
 
-const VIEW_PAGES = [overview, schema, describe];
+const VIEW_PAGES = [overview, schema, describe, access];
```

Comment:
maybe we could like

const COMMON_PAGES = [overview, access,....]

and then

const TABLE_PAGES = [...COMMON_PAGES, someOtherPage]

Reply:
It seems no use... only `overview` is always first page. Others are mixed for different views.